test/resources/validate/dockerfile-from-ce-image/asset.yaml

```diff
@@ -0,0 +1,11 @@
+name: sklearn-1.0
+version: auto
+type: environment
+spec: spec.yaml
+extra_config: environment.yaml
+test:
+  pytest:
+    enabled: true 
+    pip_requirements: tests/requirements.txt
+    tests_dir: tests
```

Comment:
Please remove the test section. It's not needed, nor do these files actually exist in the resources dir.